pkg/products/threescale/reconciler.go

```diff
@@ -1698,47 +1698,47 @@ func (r *Reconciler) reconcileServiceDiscovery(ctx context.Context, serverClient
 func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation *integreatlyv1alpha1.RHMI, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
 	cfg, err := r.ConfigManager.ReadMonitoring()
 	if err != nil {
-		return integreatlyv1alpha1.PhaseInProgress, fmt.Errorf("error reading monitoring config: %w", err)
+		return integreatlyv1alpha1.PhaseInProgress, nil
 	}
 
 	err = monitoring.CreateBlackboxTarget(ctx, "integreatly-3scale-admin-ui", monitoringv1alpha1.BlackboxtargetData{
 		Url:     r.Config.GetHost() + "/" + r.Config.GetBlackboxTargetPathForAdminUI(),
 		Service: "3scale-admin-ui",
 	}, cfg, installation, client)
 	if err != nil {
-		r.log.Error("Error creating threescale blackbox target", err)
-		return integreatlyv1alpha1.PhaseInProgress, fmt.Errorf("error creating threescale blackbox target: %w", err)
+		r.log.Info("Error creating threescale blackbox target: " + err.Error())
+		return integreatlyv1alpha1.PhaseInProgress, nil
```

Comment:
We should still return the error message and last few changes should still return the error messages

Reply:
In this particular case, if I'm not wrong, `err` means "not yet, just waiting for the thing" (or could mean), so, technically, it is expected (?). On a conceptual level, this ticket was not about the reduction of the severity of errors but about making severity consistent withing the codebase. If this to be changed to `warning` it means that in every other place where we saying "not ready yet, just waiting` it also should be a warning (at this moment it is `Info` everywhere). Would like to highlight, that there was a brilliant idea of making `event.HandleError` do the thing for us: always return an error and make sure we always call `event.HandleError` and there we can unify the way we treat each of the states (for example, always `log.Warning` for the `PhaseInProgress`

Reply:
Well it depends on what caused the `err`. To me the core function used would only return an error if there was an issue in creating the `BlackBoxTarget` so it's not waiting anything if an error is returned :thinking: 
* https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/monitoring/reconciler.go#L816-L864

This to me is at least a warning since it's not expected or should happen frequently. Again referencing my main review comment, maybe `integreatlyv1alpha1.PhaseInProgress` is not the right phase to return here if there is an error, we should have a conversation with CS SRE on whether this loss of propagation of the error to the CR `lastError` field is okay on the off chance the reconciler is stuck at this point for some reason :thinking: